New Jobs/War_Correspondent.txt

```diff
@@ -0,0 +1,26 @@
+## Abstract
+
+War Correspondent is a journalist role, meant to be a sort've independent/marine aligned CL. It's meant to provide more roleplay opportunities and add more flavour to the game I guess.
+
+## Goals
+
+1. Create a role that's interesting and provides new oppirtunities along with spicing up the game I suppose
+2. Create a roleplay-centric role that's differentiated from the CL enough to be its own thing
```

Comment:
what roleplay opportunities is this role supposed to bring. Detail is everything, avoid over-generalisation